tests/test_dynamic_membership.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+### This script tests dynamic membership by starting a Raft network with 3 nodes, adding a 4th node, applying a workload, and checking that all 4 nodes reach block 5
+
+echo "Starting initial network"
+docker-compose -f ../adhoc/admin.yaml up -d
+docker-compose -p alpha -f ../adhoc/node.yaml up -d
+docker-compose -p beta -f ../adhoc/node.yaml up -d
+GENESIS=1 docker-compose -p gamma -f ../adhoc/node.yaml up -d
+
+echo "Gathering list of initial keys and REST APIs"
+INIT_KEYS=($(docker exec $(docker ps | grep admin | awk '{print $1;}') bash -c 'cd /shared_data/validators && paste $(ls -1) -d , | sed s/,/\ /g'))
```

Comment:
You can assign `$(docker ps | grep admin | awk '{print $1;}')` to a variable so you don't need to rerun the command constantly, as the output shouldn't change.